scripts/test.sh

```diff
@@ -5,12 +5,12 @@ set -e -x
 run () {
   case $CIRCLE_NODE_INDEX in
   0)
-    yarn acceptance
-    yarn mocha test/lib/*
-    yarn mocha $(find desktop/test -name '*.coffee')
-    yarn mocha $(find mobile/test -name '*.coffee')
+    yarn assets
+    yarn acceptance test/acceptance/*.js
```

Comment:
Letting acceptance tests take up one container for itself now which evens out the build time for now.